rubocop/README.md

```diff
@@ -0,0 +1,36 @@
+# How to use Rubocop configs
+Reference the rubocop config that you want from another repo by inheriting from this file. When you are ready to upgrade rubocop to a different version, change the file you `inherit_from` in your repo's local `.rubocop.yml`.
+
+Here is an example of what your repo's local `.rubocop.yml` will look like
+```
+inherit_from: https://raw.githubusercontent.com/q-centrix/style-guides/main/rubocop/.rubocop-0-77.yml
+```
+
+Note: Anything you add to your local yml file will overwrite what is in the inherited file. Read [the docs](https://docs.rubocop.org/rubocop/1.50/configuration.html#inheriting-from-another-configuration-file-in-the-project) for more information on precedence. This can be helpful if you want to add in something such as rubocop-rspec but want to disable/enable cops as you fix them over time (see [auto gen configs](https://docs.rubocop.org/rubocop/configuration.html#automatically-generated-configuration)).
+
+### Other places you will need to check
+#### Only the first time
+1. `.codeclimate.yml`
+    1. add a config file to the plugin so it will find the file in the correct place
+        ```
+        rubocop:
+          enabled: true
+          channel: rubocop-0-77
```

Comment:
Should this `channel` setting be in the `With every update` section?  Seems like this would change on every update.

Reply:
yes, good catch! Not sure how I missed that 😅

Reply:
It is there already, haha, see 1.2 below. But I'll add a code snippet to make it harder to miss